### PR TITLE
Archiving the tests results even if the downstream job fails 

### PR DESCRIPTION
**Pull request by arindam-bandyopadhyay**

### CODE REVIEW

**Comment by arindam-bandyopadhyay** [comment]
@glassfishrobot Run CI tests please

File: appserver/tests/quicklook/run_test.sh [hunk 1]
```diff
@@ -90,6 +90,19 @@ run_test_id(){
...
 	fi
     change_junit_report_class_names
+}
+
+post_test_run(){
+    if [[ $? -ne 0 ]]; then
+    	if [[ $TEST_ID = "ql_gf_full_profile_all" || $TEST_ID = "ql_gf_web_profile_all" || $TEST_ID = "ql_gf_embedded_profile_all" ]]; then
+	  		copy_ql_results || true
+	  	fi
+	  	if [[ $TEST_ID = "ql_gf_nucleus_all" || $TEST_ID = "nucleus_admin_all" ]]; then
```
Comment (yaminikb) [inline appserver/tests/quicklook/run_test.sh]: This is a QL run script, why are we checking for nucleus_admin_all?

Comment (arindam-bandyopadhyay) [inline appserver/tests/quicklook/run_test.sh]: This run_test.sh is wrapper for the following test ids 
list_test_ids(){
	echo ql_gf_full_profile_all ql_gf_nucleus_all ql_gf_web_profile_all ql_gf_embedded_profile_all nucleus_admin_all
}

Comment (yaminikb) [inline appserver/tests/quicklook/run_test.sh]: I see. Thanks for clarifying.

**Review by yaminikb — APPROVED** [review]

**Review by sameer-pandit — APPROVED** [review]
Looks good.

**Comment by yaminikb** [comment]
Merged after verifying that the one test failure was an intermittent failure in web devtests.

**Pull request merged**